overlord/assertstate/bulk.go

```diff
@@ -175,7 +175,8 @@ func bulkRefreshValidationSetAsserts(s *state.State, vsets map[string]*Validatio
 		}
 	}
 
-	err := resolvePoolNoFallback(s, pool, beforeCommitChecker, userID, deviceCtx)
+	opts := &RefreshSnapDeclarationsOptions{IsAutoRefresh: true}
```

Comment:
same question do we check that this option is specifically set to true in any test?

Reply:
I misread this, this should come from outside through the levels for symmetry and also because some callers in the end will not be from auto-refresh I think